### DISCUSSION TITLE
Suricata Version

### DISCUSSION THREAD

**Discussion by fogel-j** [Security-Onion-Solutions/securityonion#13452] [General]
Where can I check the suricata version that's running on my security onion system?

**Comment by dougburks** [comment]
There are a few ways you can check the Suricata version on your system.

You can ask Suricata to display its version:
```
sudo docker exec so-suricata /opt/suricata/bin/suricata -V
This is Suricata version 7.0.6 RELEASE
```

Or you can check the first line of the Suricata log file:
```
sudo head -1 /opt/so/log/suricata/suricata.log 
[11 - Suricata-Main] 2024-08-09 07:05:52 Notice: suricata: This is Suricata version 7.0.6 RELEASE running in SYSTEM mode
```

You can also see when Suricata updates were included in Security Onion releases via our Release Notes:
https://docs.securityonion.net/en/2.4/release-notes.html
